### PR TITLE
add php version logging, remove multilingual press notice

### PR DESCRIPTION
**Pull request by vsolovei-smartling**

### CODE REVIEW

File: inc/Smartling/Bootstrap.php [hunk 1]
```diff
@@ -205,17 +205,8 @@ public function testMultilingualPressPlugin()
...
-
         if (class_exists('Mlp_Load_Controller', false)) {
             $mlPluginsStatuses['multilingual-press-pro'] = true;
-            $found = true;
-        }
-
-        if (false === $found) {
-            add_action('admin_init', function () {
-                DiagnosticsHelper::addDiagnosticsMessage('Recommended plugin <strong>Multilingual Press</strong> not found. Please install and activate it.', false);
-            });
```
Comment (PavelLoparev) [inline inc/Smartling/Bootstrap.php:218]: Do we need `testMultilingualPressPlugin` function at all since Multilingual Press plugin is not a dependency?

Comment (vsolovei-smartling) [reply inc/Smartling/Bootstrap.php:218]: Yes, I've renamed the function to better indicate intent, it's result is being used later to improve compatibility with MultilingualPress if it installed.

File: inc/Smartling/DITrait.php [hunk 2]
```diff
@@ -68,7 +68,7 @@ protected static function initContainer()
         $host = false === gethostname() ? 'unknown' : gethostname();
         LogContextMixinHelper::addToContext('host', $host);
         LogContextMixinHelper::addToContext('http_host', $_SERVER['HTTP_HOST']);
-        LogContextMixinHelper::addToContext('moduleVersion', static::$pluginVersion);
```
[the remark this answers is not in the document]
Comment (vsolovei-smartling) [reply inc/Smartling/DITrait.php:71]: Renamed. Php version is being sent in user agent currently, so far I've seen only one projectId with plugin version 1.10 that uses php 5.

**Review by PavelLoparev — APPROVED** [review]

File: inc/Smartling/DITrait.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (dimitrystd) [reply inc/Smartling/DITrait.php:71]: +1 let's do it.

**Review by dimitrystd — APPROVED** [review]

**Pull request merged**